### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by thgreasi**
This forks & modifies `whatwg-fetch` to work as a ponyfill on the Google Apps Script environment  and publishes it as `fetch-google-apps-script-ponyfill`.

Change-type: minor
See: https://www.flowdock.com/app/rulemotion/p-billing/threads/cNz2yByrqr3g-5QDsusLJAY-RwZ
See: https://www.flowdock.com/app/rulemotion/resin-frontend/threads/KCe7WBxlAR7MEDyxZpJFo7skczB
See: https://developers.google.com/apps-script/reference/url-fetch

### CODE REVIEW

File: fetch.js [hunk 1]
```diff
@@ -424,93 +408,32 @@ Response.redirect = function (url, status) {
...
-      xhr.responseType = 'blob'
+  return Promise.resolve().then(function () {
+    if (init === undefined || init.muteHttpExceptions == null) {
+      init = Object.assign({}, defaults, init)
     }
 
-    request.headers.forEach(function (value, name) {
-      xhr.setRequestHeader(name, value)
-    })
+    var urlFetchAppRequest = UrlFetchApp.fetch(input, init)
```
Comment (iamsolankiamit) [inline fetch.js:422]: What is `UrlFetchApp`? I see that it is a global object, wouldn't it be better if it was imported instead?

Comment (iamsolankiamit) [reply fetch.js:422]: Got it, https://developers.google.com/apps-script/reference/url-fetch/url-fetch-app. LGTM now.

**Review by iamsolankiamit — APPROVED** [review]

File: fetch.js (hunk 1 above)
Comment (thgreasi) [reply fetch.js:422]: :)

Comment (thgreasi) [reply fetch.js:422]: I always miss checking the description in PRs as well :)

**Pull request merged**